packages/api/src/auth/decoders/azureAd.ts

```diff
@@ -0,0 +1,7 @@
+import jwt from 'jsonwebtoken'
+
+export const azureAd = (token: string) => {
+  // @todo: JWT validation
```

Comment:
From reading some [Azure AD docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/access-tokens#validating-the-signature), the verification could be similar the Auth0's which is implemented by getting the signing key via a jwksClient:

https://github.com/redwoodjs/redwood/blob/main/packages/api/src/auth/decoders/auth0.ts#L25

Reply:
Thanks @dthyresson! I'll have a closer look later this week :)
Also, I have marked the PR's as a drafts to make life easier for maintainers.

Reply:
@dthyresson I will mark this as resolved (as the new file is called `azureActiveDirectory`) which contains validation. I'll ping you in a new reply.